build/cdc/dev/docker-compose.yaml

```diff
@@ -0,0 +1,125 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Docker compose for running a local development version of Data Commons
+# Prerequisites:
+#   - docker
+#   - Copy .env.sample to .env and fill in missing values
+# Run with ./run_cdc_dev_docker.sh
+
+services:
+  dc-mixer:
+    build:
+      context: ../../../
+      dockerfile: build/cdc/dev/Dockerfile.mixer.cdc.dev
+    container_name: dc-mixer
+    env_file:
+      - .env
+    environment:
+      - MAPS_API_KEY
+      - MIXER_API_KEY=$DC_API_KEY
+      - GOOGLE_APPLICATION_CREDENTIALS=/app/creds.json
+    ports:
+      - "8081:8081"
+    networks:
+      - webnet
+    volumes:
+      - ../../../mixer:/app/mixer
+      - ./run_mixer.sh:/app/run_mixer.sh
+      # GCP credentials
+      - "${HOST_GOOGLE_APPLICATION_CREDENTIALS:-${HOME}/.config/gcloud/application_default_credentials.json}:/app/creds.json"
+
+  # Website frontend static assets
+  dc-website-static:
+    build:
+      context: ../../../
+      dockerfile: build/cdc/dev/Dockerfile.website-static.cdc.dev
+    container_name: dc-website-static
+    volumes:
+      # Website frontend source code
+      - ../../../static:/app/static
+      # Website frontend dependencies
+      - ../../../packages:/app/packages
+      # Statically compiled files
+      - ../../../server/dist:/app/server/dist
+
+  # Website flask API
+  dc-website:
+    build:
+      context: ../../../
+      dockerfile: build/cdc/dev/Dockerfile.website.cdc.dev
+      network: host
+    container_name: dc-website
+    env_file:
+      - .env
+    environment:
+      - DC_API_KEY
+      - MAPS_API_KEY
+      - ENABLE_MODEL=true
+      - FLASK_ENV=local
+      - GOOGLE_CLOUD_PROJECT=datcom-website-dev
+      - GOOGLE_APPLICATION_CREDENTIALS=/app/creds.json
+      - NL_SERVICE_ROOT_URL=http://dc-nl-python:6060
+      - WEBSITE_MIXER_API_ROOT=http://dc-mixer:8081
```

Comment:
Can you set this variable in [.run_cdc_dev.env](https://github.com/datacommonsorg/website/blob/7f896a982e8567cd96a0d8b01d1cd5eaaf285974/.run_cdc_dev.env#L53) as well?

Reply:
Done